sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -328,7 +328,8 @@ nullNotnull
     ;
 
 functionCall
-    : scalarFunctionName LR_BRACKET functionArgs RR_BRACKET         #scalarFunctionCall
+    : nestedFunctionName LR_BRACKET allTupleFields RR_BRACKET       #nestedAllFunctionCall
```

Comment:
This is fine for now, but if we get any more functions that need to behave like this (`highlight`?) then we will need to refactor to something like `scalarFunctionName`.

Reply:
Yes the purpose is to only support `allTupleFields` with the nested function.